### DISCUSSION TITLE
Has Now been renamed to Vercel?

### DISCUSSION THREAD

**Discussion by thgh** [vercel/vercel#4468] [General] [answered]
The announcement mentioned that Zeit was renamed to Vercel, but in a lot of places Now was also renamed to Vercel (cli, website, ...).

Will `now` continue to be used? Should a builder like `now-sapper` be renamed to `vercel-sapper`?

**Comment by TooTallNate** [comment, marked as answer]
Yes, the Now name has become Vercel as well.

> Will `now` continue to be used?

For the npm package name, `now` will be co-published along with `vercel` for backwards-compatibility purposes. On Jan 1, 2021 we will stop publishing to the `now` package name.

I suggest that you switch over to the `vercel` package name at your convenience. Note that you can use the shortname `vc` command name to invoke the CLI (one character shorter than `now`!)

> Should a builder like `now-sapper` be renamed to `vercel-sapper`?

Yes. For example, my Bash builder has been [renamed to `vercel-bash`](https://github.com/importpw/vercel-bash/commit/51316b25553568e61c4467a00bfd09a546c35c5b).

**Reply by ninest** [reply]
Does that mean that if we have a site hosted on `name.now.sh`, it will change to `name.vercel.com` or `name.vercel.sh`?

Edit based on the Notion link below: It seems it will change to `name.vercel.app`. Will there be any way to keep `name.now.sh`?

**Comment by nattstack** [comment]
Here is a notion link for the change timeline: https://www.notion.so/Change-Timeline-fab7622780694753a92c0a396cea7eb6
